backend/database/flower.go

```diff
@@ -9,12 +9,13 @@ import (
 )
 
 type Flower struct {
-	ID        ObjectID  `json:"_id,omitempty" bson:"_id,omitempty"`
-	Name      string    `json:"name"`
-	LatinName string    `json:"latin_name" bson:"latin_name"`
-	AddedTime time.Time `json:"added_time" bson:"added_time"`
-	Grower    *ObjectID `json:"grower"`
-	Site      *ObjectID `json:"site"`
+	ID          ObjectID  `json:"_id,omitempty" bson:"_id,omitempty"`
+	Name        string    `json:"name"`
+	LatinName   string    `json:"latin_name" bson:"latin_name"`
+	AddedTime   time.Time `json:"added_time" bson:"added_time"`
+	Grower      *ObjectID `json:"grower"`
+	GrowerEmail string    `json:"grower_email" bson:"grower_email"`
```

Comment:
That’s a very good point! Truthfully i did not even consider what would happen in case the email (or whatever else contact info the grower wants provided in that field in the future) has to be changed. I found this approach to be the simplest way to get the email to be displayed on the frontend, since all i had to do was update AddFlower-function in backend and insert { flower.grower_email } inside <td></td> tags in frontend. I’m definitely open to suggestion on how you’d approach this differently! Perhaps by modifying the GetFlowers-function in the backend to return the email from User’s document along with the flowers? 

Do you think this should be changed before this PR can be merged, or is it something to be done in the future?

Reply:
I do not think this needs to be changed now. It works well and it is simple. We can think about in the future, but I still wanted to point it out. If we want to allow contact info updating in the future, we can worry about it then.

The way I would approach this is that email is not stored in the flower by default and it is not fetched for the retailer view by default, but maybe when user is explicitly asking for the contact information after they have reviewed the flower and want to order it. However this is not something in the scope of current user story, unlike the way retailer works in your pull request.